go/tools/gazelle/resolve/resolve.go

```diff
@@ -77,6 +77,12 @@ func NewLabelResolver(c *config.Config) LabelResolver {
 	switch c.DepMode {
 	case config.ExternalMode:
 		e = newExternalResolver()
+	case config.KnownMode:
+		er := newExternalResolver()
```

Comment:
That's a good question. Let's keep them in resolve_external.go for now. Config is created in a lot of places for tests, so that would make this change more complicated. I may refactor it later.